lua/core/audio.lua

```diff
@@ -208,6 +209,209 @@ Audio.level_cut_rev = function(value)
 end
 
 
+
+--- Routing Functions
+-- @section Routing
+
+--- default audio connections for softcut
+Audio.default_softcut_routes = {
+  ["softcut:output_1"] = "crone:input_3",
+  ["softcut:output_2"] = "crone:input_4",
+  ["crone:output_3"]   = "softcut:input_1",
+  ["crone:output_4"]   = "softcut:input_2",
+}
+
+--- default audio connections for supercollider
+Audio.default_supercollider_routes = {
+  ["SuperCollider:out_1"] = "crone:input_5",
+  ["SuperCollider:out_2"] = "crone:input_6",
+  ["crone:output_5"]      = "SuperCollider:in_1",
+  ["crone:output_6"]      = "SuperCollider:in_2",
+}
+
+--- default audio connections for system (hardware)
+Audio.default_system_routes = {
+  ["system:capture_1"] = "crone:input_1",
+  ["system:capture_2"] = "crone:input_2",
+  ["crone:output_1"]   = "system:playback_1",
+  ["crone:output_2"]   = "system:playback_2",
+}
+
+Audio.default_system_routes_reversed = {
```

Comment:
rather than keep a separate table for reversed routes, would it be bad form to just modify the default routes table in-place? that way we skip all of the extra checking and comparisons later? (by this i mean, the routes table gets modified upon detection of the reverse.txt at startup)

but as-is has the advantage of being very clear and less error prone

Reply:
that is a totally fair observation. one vague thought i had around the tables was tweaking them in a mod such that an alternative "default" configuration was setup by the system w/o additional code but i couldn't decide on how to do that while also staying compatible with the reverse stereo case. the one concern i have about programmatically modifying them is that it implies the jack client names and routings for the "system" paths are fixed. 

fwiw an earlier version of this didn't expose the tables at all.

Reply:
mmmm i sortof like the idea of a MOD just to flip the channels, given this is just a narrow batch of shields affected (and, curiously, many people don't seem to notice?) ---- just to keep the core codebase small

Reply:
a mod sounds like an interesting path, will revise this PR.